src/driver/driver_api.cc

```diff
@@ -225,6 +225,7 @@ Array<tvm::transform::Pass> CreatePassList(bool disable_loop_partition) {
   }
   // LowerVtcmAlloc must occur after any transformations that modify memory allocation locations
   pass_list.push_back(tir::transform::LowerVtcmAlloc());
+  pass_list.push_back(tir::transform::LowerAsyncDMA());
```

Comment:
May need to have a device specific (Hexagon) enable here and/or TIR annotation or some other mechanism to enable this pass to avoid breaking existing async pipeline implementations e.g. cuda.

Reply:
Resolving this comment.  Will track this in conversation on the [comment](https://github.com/apache/tvm/pull/12785#discussion_r971701650) from @masahi as it is related.